string  vehicle_name
string  vehicle_detect
string  status
string  map_to_use
string  coupling_or_decoupling

uint16  id0
uint16  id1
uint16  id2
uint16  id_marker
uint16  num_of_initial_agv

float32 x_point
float32 y_point
float32 theta

